sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -1934,6 +1934,50 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
     }
   }
 
+  test("Star Expansion - CreateStruct and CreateArray") {
```

Comment:
Why do we put these tests in `SQLQuerySuite`? It looks like they are mostly testing DF APIs.